### ISSUE TITLE
Version >=3.6.0 fails to parse RegEx object values

### ISSUE THREAD

**Issue by tobi-fis** [josdejong/jsonrepair#123]
Hey everyone,

we noticed that from 3.6.0 upwards, jsonrepair runs into an error when feeding it with a config file we are using.
In this config file, one of the object values is a RegEx Pattern, as this is expected by the framework (Vitest) which uses this config.
The raw, 'unrepaired' json looks something like this:
```
       {
            ...
            foo: ['default', 'some-other-reporter'],
            bar: './some/file/location.xml',
            baz: ['./another/file/path.ts'],
            css: { 
                      include: /standalone-styles.css/   // this fails starting from 3.6.0
                  },
            ...
        },
```
Adding quotes around `/standalone-styles.css/` is not possible, as this would break the RegEx. So it has to be written as it is.

I tested this with several versions:

- **Unaffected**: 3.1.0 - 3.5.0
- **Affected**: 3.6.0, 3.6.1

The error we get is ` Unexpected character "/" at position 359`

For now, we just refrain from using newer versions. Hope to see a fix for this, though.

Thanks in advance and best of regards,
Tobias

**Comment by josdejong** [comment]
Thanks for reporting! 

I wasn't aware that the library was able to repair regular expressions in the first place 😅. 

Fixed via c475377be0d722b1c578314204c488970d4e1746 (not yet published).

**Comment by josdejong** [comment]
Fixed now in `v3.7.0`
